lib/bindConnectors/formatError.js

```diff
@@ -18,11 +22,24 @@ module.exports = function (response, event) {
 
 	if (_.isError(details)) {
 
-		details = {
+		var newDetailsObject = {
 			message: details.message || 'API error',
-			code: 'api_error'
+			code: details.code || '#api_error'
 		};
-		
+
+		switch (details.constructor) {
+			case TypeError:
+			case ReferenceError:
+			case SyntaxError:
+				if (details.stack) {
+					newDetailsObject.stack = cleanStack(details.stack);
+					newDetailsObject.code = details.code || '#connector_error';
```

Comment:
This line should rather be outside of the if statement, as its independent of the stack